trento/xml/article_sap_trento.xml

```diff
@@ -2336,9 +2347,16 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
 
       <section>
         <title>Scenario dump</title>
-        <para>The scenario dump is a dump of the Trento database. It helps the core team
-          to recreate a mock environment with the same status as the customer's.
-          To generate the dump, proceed as follows:</para>
+        <para>The scenario dump is a dump of the Trento database. It helps the Trento team
+          to recreate a mock environment with the same status as the customer's. Using the script
+        that generates the dump requires a host with the following setup:</para>
```

Comment:
After reading the sentence, maybe we should simplify the sentence? I assume, the customer doesn't care about the details why the Trento team needs it. Your addition in the last sentence should be preserved, of course:

```suggestion
        <para>The scenario dump is a dump of the Trento database. It helps the Trento team
          to recreate the scenario to test it. Using the script
        that generates the dump requires a host with the following setup:</para>
```